Classes/Console/Install/StepsConfig.php

```diff
@@ -31,6 +31,9 @@ public function __construct(string $baseConfigFile = null)
 
     public function getInstallSteps(string $stepsConfigFile = null): array
     {
+        if ($stepsConfigFile === null && getenv('TYPO3_INSTALL_SETUP_STEPS')) {
+            $stepsConfigFile = realpath(getenv('TYPO3_INSTALL_SETUP_STEPS')) ?: null;
```

Comment:
How about always using `realpath()` on the file right before the `return` and using a chained list of options here:

    $stepsConfigFile = $stepsConfigFile ?: getenv('TYPO3_INSTALL_SETUP_STEPS') ?: $this->baseConfigFile;
    $stepsConfigFile = realpath($stepsConfigFile);

Reply:
Thanks, fixed